doc/source/user_guide/session/launching_ansys_fluent.rst

```diff
@@ -300,6 +300,56 @@ can be launched explicitly using :func:`create_launcher() <ansys.fluent.core.lau
   >>> pim_solver_session = pim_solver_launcher()
 
 
+Launching a Fluent Session in Container Mode Using Docker and Podman Compose
+----------------------------------------------------------------------------
+
+You can easily launch an Ansys Fluent session in container mode by leveraging Docker and Podman Compose with PyFluent. 
```

Comment:
Docker 'or' Podman?? @hpohekar

Reply:
We are supporting both.

Reply:
@hpohekar: @prmukherj was proposing a change to the wording of the heading. I don't know which is better, but with "or" it might be less ambiguous(?) 

Meanwhile:
- The heading is fully capitalized (Session, Container, Mode, Using) which we don't do elsewhere because of the PyAnsys convention.
- We should write **Docker Compose** fully to be clear what we mean.

cc: @mkundu1

Reply:
"Easily" is superfluous here. It's not obvious what we're comparing against.

Reply:
@hpohekar @prmukherj @mkundu1 Heading: "Launching Fluent with Docker Compose or Podman Compose"

Reply:
Done. Thank you. I thought we are supporting both therefore I used and earlier but we have changed it to or now. Title heading has changed. We have used Docker Compose at all places.